crates/compiler/src/polyfill.rs

```diff
@@ -1239,6 +1237,6 @@ mod test {
     fn has_correct_polyfill_count() {
         let polyfills = PolyfillMap::new();
         let count = polyfills.iter().count();
-        assert_eq!(count, 842);
+        assert_eq!(count, 952);
```

Comment:
I feel like there needs to be a note at the top mentioning to modify this if we add polyfills.